bigquery/snippets/snippet.go

```diff
@@ -831,7 +831,7 @@ func updateModelDescription(client *bigquery.Client, datasetID, modelID string)
 	model := client.Dataset(datasetID).Model(modelID)
 	oldMeta, err := model.Metadata(ctx)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get model metadata: %v", err)
```

Comment:
Usually just `return fmt.Errorf("Metadata: %v", err)`.

Same with other errors returned in this PR.